modules/email/pom.xml

```diff
@@ -32,8 +32,8 @@
     <properties>
         <email.service.enabled>true</email.service.enabled>
         <email.service.class.name>com.intuit.wasabi.email.impl.NoopEmailImpl</email.service.class.name>
-        <email.host>mail.intuit.com</email.host>
-        <email.from>no-reply-wasabi@intuit.com</email.from>
+        <email.service.host>localhost</email.service.host>
+        <email.service.from>no-reply-wasabi@intuit.com</email.service.from>
```

Comment:
I am not sure if this should be @intuit.com, maybe we should better use example.com?